source/connect.txt

```diff
@@ -204,6 +204,15 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to A Different Deployment
+---------------------------------
+
+You can use the :method:`Mongo()` or the :method:`connect()` methods to 
+connect to a different MongoDB deployment from within the {+mdb-shell+}.
+
+To learn how to connect to a different deployment from within the
+{+mdb-shell+}, see :ref:`mdb-shell-open-new-connections-in-shell`.
```

Comment:
Suggestion to avoid repetition: Maybe, "To learn how to connect to a different deployment using these methods..."